src/components/button-minor/button-minor.style.ts

```diff
@@ -55,24 +53,24 @@ const StyledButtonMinor = styled(Button)`
     `}
     `}
 
-  ${({ size }) => css`
+  ${({ size, iconTooltipMessage }) => css`
     ${size === "small" &&
     `
         min-height: 32px;
-        padding: var(--spacing000) var(--spacing100) var(--spacing000)
-          var(--spacing100);
+        padding-left: ${iconTooltipMessage ? "none" : "var(--spacing100)"};
```

Comment:
I actually don't like this approach at all. We shouldn't have to set different styles depending on whether an `iconTooltipMessage` prop is passed in or not - after all, just setting the prop doesn't force the tooltip to appear, as you also have to hover over the button.

Since it seems that this left and right padding is never actually needed, I'd prefer to just remove the styles entirely (assuming that doesn't cause any visual changes - if it breaks tests we can change those, as you've already done).

I've just been looking to see what is causing the icon to jump when the `<span data-portal-entrace>` appears - I haven't figured it out yet but I think it's something to do with the flex styles on the parent button. It's weird that it jumps verticall when *left* and *right* padding are added to the <span> with the icon in it - especially as it only happens when you add both, not just one. And of course the additional child, the portal-entrance span, has a width and height of 0. I'm happy to help out with this if you want me to - but I also think simply removing the padding in all circumstances is a good enough solution unless I've missed some reason why it's sometimes needed. I definitely don't like conditionally removing it only when there's potential for a tooltip to be shown.

Reply:
I agree, I've went for an absolute position approach on the rendered `Icon` opposed to the conditional padding approach 👍